<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkSpaceClass</class>
 <widget class="QMainWindow" name="WorkSpaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkSpace</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="rightMargin">
      <number>500</number>
     </property>
     <property name="bottomMargin">
      <number>400</number>
     </property>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOpen">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset resource="workspace.qrc">
         <normaloff>:/WorkSpace/Resources/icons/newScene.png</normaloff>:/WorkSpace/Resources/icons/newScene.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="workspace.qrc">
         <normaloff>:/WorkSpace/Resources/icons/export.png</normaloff>:/WorkSpace/Resources/icons/export.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mClear">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="workspace.qrc">
         <normaloff>:/WorkSpace/Resources/icons/Eraser.png</normaloff>:/WorkSpace/Resources/icons/Eraser.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mSearch">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset resource="workspace.qrc">
         <normaloff>:/WorkSpace/Resources/icons/Search-icon.png</normaloff>:/WorkSpace/Resources/icons/Search-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>541</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QWidget" name="thumbnailWidget_2" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="thumbLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QScrollBar" name="scrollBar_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="actionGenData"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="workspace.qrc">
     <normaloff>:/WorkSpace/Resources/icons/newScene.png</normaloff>:/WorkSpace/Resources/icons/newScene.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="workspace.qrc">
     <normaloff>:/WorkSpace/Resources/icons/export.png</normaloff>:/WorkSpace/Resources/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGenData">
   <property name="text">
    <string>GenData</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="workspace.qrc"/>
 </resources>
 <connections/>
</ui>
